website_mail_snippet_fixed/static/src/css/fixed_560px.css

```diff
@@ -0,0 +1,22 @@
+/* ---- thumbnail ----  */
+.oe_snippet.o_snippet_1_col .oe_snippet_thumbnail_img {
+    background-image: url("/website_mail_snippet_fixed/static/src/img/snippet_1_col.png");
```

Comment:
I did as is done in mass_mailing module: 
https://github.com/OCA/OCB/blob/9.0/addons/mass_mailing/static/src/css/mass_mailing_editor_thumb.css
No other reason.